database/migrations/2023_09_05_041756_create_boards_table.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('boards', function (Blueprint $table) {
+            $table->id();
+            $table->string('uuid', 12);
+            $table->string('name', 100);
+            $table->string('ip', 15)->nullable(); //Just IPv4
+            $table->dateTime('last_seen')->nullable();
+            $table->boolean('transmitting')->default(false);
```

Comment:
Perhaps it is better to name it as working?

Reply:
It will renamed to working